Dockerfile.konflux.cuda

```diff
@@ -1,9 +1,18 @@
 ## Global Args #################################################################
 ARG RHAIIS_VERSION=3.2.2
 
+
 ## Base Layer ##################################################################
 FROM registry.redhat.io/rhaiis/vllm-cuda-rhel9:${RHAIIS_VERSION} as vllm-grpc-adapter
 
+ARG VLLM_TGIS_ADAPTER_VERSION="0.9.0"
+USER root
+
+## Install vLLM-TGIS-Adapter ####################################################
+RUN pip install --no-cache --no-deps vllm-tgis-adapter==${VLLM_TGIS_ADAPTER_VERSION}
+
+USER 2000
```

Comment:
Default user for the rhaiis image is 1001, you can check this by adding an `id -a` command before switching user:

```
#6 [2/3] RUN id -a
#6 0.255 uid=1001(default) gid=0(root) groups=0(root)
```